app/src/main/java/com/example/tasky/agenda_details/domain/repository/AgendaDetailsRemoteRepository.kt

```diff
@@ -16,4 +18,12 @@ interface AgendaDetailsRemoteRepository {
     suspend fun deleteEvent(): Result<Unit, DataError.Network>
 
     suspend fun updateEvent(): Result<EventResponse, DataError.Network>
+
+    suspend fun createTask(
+        taskDetails: TaskResponse
```

Comment:
To extend my previous PR's feedback, the response here should also just be in data and you pass the domain model of `AgendaItem.Task` instead 👍 

If your model ends with Response, Request, DTO, Entity or similar, that's 100% a data model since it just represents how data is transferred/persistently saved